engine/language-server/src/main/scala/org/enso/languageserver/boot/MainModule.scala

```diff
@@ -316,6 +316,11 @@ class MainModule(serverConfig: LanguageServerConfig, logLevel: Level) {
     RuntimeOptions.JOB_PARALLELISM,
     Runtime.getRuntime.availableProcessors().toString
   )
+  if (System.getProperty("enso.dev.inspectPort") != null) {
+    val inspectPort = Integer.getInteger("enso.dev.inspectPort", 34567);
```

Comment:
I don't think this is necessary. Just use regular `enso --inspect`:
```
    --inspect                              Start the Chrome inspector when
                                           --run is used.
```
but yeah, I am not sure if it works in "language server" mode. I'd rather got it working then designing new `enso.dev.inspectPort` property.

Reply:
Removed in 696d9e3843dfae68ce3a68af50bf4569ba951569